docs/plot.py

```diff
@@ -0,0 +1,19 @@
+"""This script plots the all-sky image following the example on getting started page"""
+import matplotlib.pyplot as plt
+from astropy.coordinates import SkyCoord
+
+from hips import make_sky_image
+from hips import HipsSurveyProperties
+from hips.utils import WCSGeometry
+
+url = 'http://alasky.unistra.fr/DSS/DSS2Merged/properties'
+hips_survey = HipsSurveyProperties.fetch(url)
+geometry = WCSGeometry.create_simple(
+    skydir=SkyCoord(0, 0, unit='deg', frame='galactic'),
+    width=2000, height=1000, fov="3 deg",
+    coordsys='galactic', projection='AIT'
+)
+data = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format='fits')
+ax = plt.subplot(projection=geometry.wcs)
+ax.imshow(data, origin='lower', cmap='gray')
+ax.grid(color='white', ls='solid')
```

Comment:
Suggest to remove the `ax.grid` call here, and to change the image scale to sqrt or something so that it's not so dark (at the moment, it looks almost all black).

See http://astropy.readthedocs.io/en/latest/api/astropy.visualization.mpl_normalize.simple_norm.html
I don't know if you can pass `norm='sqrt'` to `plt.imshow` directly, or if you have to do:
```
from astropy.visualization.mpl_normalize import simple_norm
norm = simple_norm(data, 'sqrt')
```
and then pass this norm.

Also: we could make the image a little bit smaller (e.g. 1600, 900) or something like that) so that the docs build and tests run a bit faster, but look just as nice / test just as much. If you want, you could do that change in this PR, or I or you will do it tomorrow.